app/serializers/activitypub/note_serializer.rb

```diff
@@ -164,4 +196,34 @@ def name
 
   class CustomEmojiSerializer < ActivityPub::EmojiSerializer
   end
+
+  class OptionSerializer < ActiveModel::Serializer
+    class RepliesSerializer < ActiveModel::Serializer
+      attributes :type, :total_items
+
+      def type
+        'Collection'
+      end
+
+      def total_items
+        object.votes
+      end
+    end
+
+    attributes :type, :name
+
+    has_one :replies, serializer: ActivityPub::NoteSerializer::OptionSerializer::RepliesSerializer
+
+    def type
+      'Note'
+    end
```

Comment:
I'm not sure why answer options would be Notes?

Reply:
It's in the example here: https://www.w3.org/TR/activitystreams-vocabulary/#dfn-question